xla/stream_executor/cuda/cuda_dnn.cc

```diff
@@ -5517,9 +5517,13 @@ GetCudnnFusedMHABackwardOperationGraph(
 
 #if (CUDNN_VERSION >= 8901 && TF_ENABLE_CUDNN_FRONTEND)
   // bias backward
-  TF_ASSIGN_OR_RETURN(auto tensor_dbias, CreateCudnnBiasBwdTensor(
-                                             intermediate_ops, p_dims,
-                                             p_strides, dtype, tensor_ds_mask));
+  if (use_bias) {
```

Comment:
@cheshire This was missed from previous PRs and discovered during testing. Let me know if you'd like these changes to be separated.